sale_exception/wizard/sale_exception_confirm_view.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
```suggestion
<?xml version="1.0" ?>
```

Reply:
@Tonow-c2c I don't think we should change the encoding here, it's relevant only on Python 3 files, no?